src/GrpcProtos/capture.proto

```diff
@@ -12,6 +12,20 @@ import "tracepoint.proto";
 // This is needed by LockFreeBufferCaptureEventProducer.
 option cc_enable_arenas = true;
 
+message InstrumentedFunction {
+  string file_path = 1;
+  uint64 file_offset = 2;
+  uint64 function_id = 3;
+
+  enum FunctionType {
+    kRegular = 0;
+    kTimerStart = 1;
+    kTimerStop = 2;
+  }
+  FunctionType function_type = 4;
+  bytes function_name = 5;
```

Comment:
I think we should keep using `string` until we are sure that this is not a problem. You are not gaining a lot here anyway.

Reply:
done.